### DISCUSSION TITLE
İşlemtim sistemi

### DISCUSSION THREAD

**Discussion by Musa-Sina-Ertugrul** [karatasy/ulak-avionic#8] [Ideas]
Bana dediğiniz işletim sistemi sanırım jammy jellifish ti o ubuntunun ismi ayrı bi ubuntu temelli işletim sistemi değil. Ubuntu yerine Lubuntu kullanmalıyız hatta dikte ediyorum. Aşırı Lightweight x86_64 te 2 iş parçacığı 4gb ram ile test ettim hiç kasmadı. Test ettiğim program qemu

**Comment by karatasy** [comment]
buyuk bilgisayar kullanilmasi suanlik planda yok olursa bu kisma gerin eliriz
